tests/testthat/test-run_grid.R

```diff
@@ -204,6 +210,23 @@ test_that("compare baseline works as expected", {
                         cp, 0)
 
   expect_equal(z$inc_vaccinated - z$inc_primary - z$inc_vbe, z$inc_revaccinated)
+
+  # number undergoing booster vaccination over 3 years is equal to the
+  # cumulative number for the 3 years
+
+  expect_equal(sum(z$inc_revaccinated[, ]),
+               sum(z$inc_cum_revaccinated[length(tt) - 1, ]))
+
+  # cumulative primary vaccination + cumulative booster vaccination =
+  # cumulative vaccinated overall when vbe = 0
+
+  s <- sum(z$inc_cum_primary[length(tt) - 1, ]) +
+    sum(z$inc_cum_revaccinated[length(tt) - 1, ])
+
+  t <- sum(z$inc_cum_vaccinated[length(tt) - 1, ])
+
+  expect_equal(s, t)
```

Comment:
```suggestion
expect_equal(z$inc_cum_primary + z$inc_cum_revaccinated, z$inc_cum_vaccinated)
```
This way you're testing all time points, rather than just the last one